src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -161,13 +166,18 @@ export const TradeInput = memo(() => {
   const activeSwapperName = useAppSelector(selectActiveSwapperName)
   const activeSwapperSupportsSlippage = getSwapperSupportsSlippage(activeSwapperName)
   const sortedQuotes = useAppSelector(selectSwappersApiTradeQuotes)
-  const quoteAffiliateFeeFiatPrecision = useAppSelector(selectQuoteDonationAmountUserCurrency)
   const rate = activeQuote?.steps[0].rate
 
   const isQuoteLoading = useAppSelector(selectSwappersApiTradeQuotePending)
+  const isSnapshotApiQueriesPending = useAppSelector(selectIsSnapshotApiQueriesPending)
```

Comment:
Note, snapshot `get_vp` queries take a long, long time to respond (between 5 and 10+ seconds each) so this may affect time to first swap.

We could improve this by persisting the slice for returning users (i.e get optimistic on the voting power from cached, stale data, and reconciliate with fresh data once fetched), but that would still be an issue for *first* swap without voting power persisted